spec/spec_helper_local.rb

```diff
@@ -4,7 +4,7 @@
 RSpec.configure do |config|
```

Comment:
I don't think we include this file anymore which may cause problems

Reply:
I split this when I switched to PDK. I think it used to be part of the `spec_helper.rb` before